### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by carllerche** [carllerche/tower-web#17]
Figure out the roadmap.

**Comment by lnicola** [comment]
Any updates on this? :smile: I imagine `tower-web` is on the back burner now, but it would be nice to have some kind of roadmap, especially with Tide being quite active lately.

**Comment by carllerche** [comment]
Good question.

Right now, the focus is to ship Tower. There has been a lot of activity in the [tower](http://github.com/tower-rs/tower) repository. Some of this is an effort to extract the work from here into a more reusable location.

After that, the warp / tower-web merge will start happening. I've started experimenting w/ what that could look like. I hope to report back soon, but shipping tower proper is step 1.

Does this answer your question?

**Comment by lnicola** [comment]
Yes, no worries. Especially with the upcoming futures changes, it's reasonable to work on`tokio` first, then `tower`, then `tower-web`.

**Comment by carllerche** [comment]
There will probably be another big breaking change tower-web release before
the warp merge. There are a bunch of API tweaks that I want to experiment
with before figuring out how to merge with warp.
